### ISSUE TITLE
It would be great if we have badge with number on the PROBLEMS tab

### ISSUE THREAD

**Issue by uudashr** [Microsoft/vscode#34703]
Sometimes I work with "OUTPUT" tab selected and don't realize there are problems exists.
<img width="459" alt="screen shot 2017-09-20 at 9 41 34 pm" src="https://user-images.githubusercontent.com/166053/30650391-f5d4d158-9e4c-11e7-8aef-dba57520bde6.png">

I have to switch tab all the time to know if problem exists
<img width="398" alt="screen shot 2017-09-20 at 9 41 25 pm" src="https://user-images.githubusercontent.com/166053/30650493-387f8b2e-9e4d-11e7-8e16-9e73b55ab901.png">

Provide badge number would be nice

- VSCode Version: 1.6.1
- OS Version: macOS Sierra 10.12.6

**Comment by sandy081** [comment]
Yeah.. we used to show the summary in the header before panels were tabs. After that, we removed that. This would be a UX decision for panels design.

@isidorn @bpasero @Tyriar @stevencl FYI

**Comment by isidorn** [comment]
I think adding a number badge for each panel tab makes sense. We can do a sketch and see how ti looks like.

**Comment by stevencl** [comment]
We looked at this when we first designed the horizontal panel UX. The idea we had then was that the badges on the horizontal panel would be the same as the badges we have on the activity bar. For example, something like:

![image](https://user-images.githubusercontent.com/1704059/30747997-c843b3de-9fa6-11e7-8e24-41f3ce07dd17.png)

Initially our thought was that the badges would show up when the specific tab isn't active and there is some new content inside that tab. Once the user opens the tab, the badge would disappear. This would appear to be inconsistent with the activity bar where many of the badges remain in place until the user takes some action such as committing some uncommitted changes. The reason for this apparent inconsistency is that we would probably want to use a badge to tell the user that there is some new content in the debug console for example, but the only way to remove that badge would be by the act of opening the debug console.

The problems tab is one that is closer to the viewlets in the activity bar since the user can take some action (such as fix the build errors and warnings) in order to get rid of the badge. But that doesn't hold true for the other tabs on the horizontal panel.
